wesnoth-wc.po

```diff
@@ -1538,17 +1545,18 @@ msgstr ""
 #: data/campaigns/World_Conquest/lua/game_mechanics/unittypedata.lua:99
 #: data/campaigns/World_Conquest/lua/game_mechanics/unittypedata.lua:156
 msgid "Looks like we fished some tasty help."
-msgstr ""
+msgstr "Vypadá to, že jsme chytili nějakou vítanou pochoutku."
 
 #: data/campaigns/World_Conquest/lua/game_mechanics/unittypedata.lua:56
 msgid "Need a friendly blade?"
-msgstr ""
+msgstr "Je třeba "
```

Comment:
tohle mi připadá neúplné - bez znalosti kontextu navrhuji volně
```suggestion
msgstr "Potřebuješ pomocnou ruku?"
```

Reply:
Jsem to rozepsal, pak překočil jinam a zapoměl na to.